### PR TITLE
[MM-30008] Add mmctl support for file imports

### PR DESCRIPTION
**Pull request by streamer45**
#### Summary

PR adds the missing pieces to implement the `mmctl import` command.

#### Ticket Link

https://mattermost.atlassian.net/browse/MM-30008

#### Related PRs

https://github.com/mattermost/mmctl/pull/285

#### Release Note
```release-note
NONE
```

### CODE REVIEW

**Review by isacikgoz — COMMENTED** [review]
Looks very good! But maybe we can go on a diet as well 😅

File: api4/job_local.go [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+func (api *API) InitJobLocal() {
+	api.BaseRoutes.Jobs.Handle("", api.ApiLocal(getJobs)).Methods("GET")
+	api.BaseRoutes.Jobs.Handle("", api.ApiLocal(localCreateJob)).Methods("POST")
+	api.BaseRoutes.Jobs.Handle("/{job_id:[A-Za-z0-9]+}", api.ApiLocal(getJob)).Methods("GET")
+	api.BaseRoutes.Jobs.Handle("/{job_id:[A-Za-z0-9]+}/cancel", api.ApiLocal(localCancelJob)).Methods("POST")
+	api.BaseRoutes.Jobs.Handle("/type/{job_type:[A-Za-z0-9_-]+}", api.ApiLocal(getJobsByType)).Methods("GET")
+}
+
+func localCreateJob(c *Context, w http.ResponseWriter, r *http.Request) {
```
[the remark this answers is not in the document]
Comment (streamer45) [reply api4/job_local.go:21]: That's great to know. Didn't particularly like all the repetition. Only thing is that the audit record will be the same for local and non-local.

**Review by isacikgoz — APPROVED** [review]
Nicely done! 👍

**Comment by streamer45** [comment]
Thank you @isacikgoz , great review as usual :+1:

**Review by mgdelacroix — APPROVED** [review]
LGTM! :+1: Great work @streamer45!!

**Comment by streamer45** [comment]
@lindalumitchell 

This PR does not have any user facing change nor does it alter existing functionality. It's a needed step to support the new `mmctl import` command.

**Review by lindalumitchell — APPROVED** [review]

**Pull request merged**